vietnamese/nguoivietnam/cudan9x.md

```diff
@@ -0,0 +1,3 @@
+Đôi lời tâm sự của một người việt Trẻ
+
+Một đất nước giàu mạnh bắt nguồn từ những người dân đủ trí, đức và tài. Cùng nhau rèn luyện, trước là để giúp mình, sau là để giúp đời. Bắt đầu mọi thứ từ những điều bạn đam mê, và sau đó thì đây là bước tiếp theo, bạn cần đến [đây](https://www.google.com/).
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.